pyuvdata/uvdata/tests/test_mir_parser.py

```diff
@@ -270,7 +308,15 @@ def test_mir_write_full(mir_data, tmp_path, data_type):
     assert mir_data == mir_copy
 
 
-def test_compass_flag_sphid_apply(mir_data, compass_soln_file):
+def test_compass_read_err(mir_data: MirParser, compass_soln_file):
+    with pytest.raises(ValueError, match="Cannot call read_compass_solns"):
+        mir_data.read_compass_solns(compass_soln_file)
+
+    mir_data.unload_data()
```

Comment:
Add a note here on why unloading fixes the error.

Reply:
Added in be15abab6f4548a576351a1f43d190cefd2fcd48.